Tests/MongoSwiftTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -22,14 +25,25 @@ struct UnifiedTestRunner {
         }
 
         /// If the internal client is a map of per-mongos clients, returns that map; otherwise throws an erorr.
-        func asMongosClients() throws -> [ServerAddress: MongoClient] {
+        func asMongosClients() async throws -> [ServerAddress: MongoClient] {
             guard case let .mongosClients(clientMap) = self else {
                 throw TestError(
                     message: "Runner unexpectedly did not create per-mongos clients"
                 )
             }
             return clientMap
         }
+
+        func closeClients() throws {
```

Comment:
nit: can we just name this `close()`? this type is meant to abstract over the number of clients so it won't always be plural clients.
also, can we add a docstring please, since all the other methods and properties on this types have one